public/client.html

```diff
@@ -39,7 +39,6 @@
 	<link type="text/css" rel="stylesheet"
 		href="https://golden-layout.com/files/latest/css/goldenlayout-dark-theme.css" />
 	<script src="lib/jdataview.min.js"></script>
-	<script src="lib/gify.min.js"></script>
 	<script src="ui.b.js"></script>
 	<script src="client.b.js?v=1.0.2"></script>
```

Comment:
```suggestion
```
Remove both of these because they're not needed!
![image](https://user-images.githubusercontent.com/36182383/158709233-c50a7e0d-477b-4786-a8da-bb4d3aa860a1.png)